src/components/TextBox/index.tsx

```diff
@@ -618,6 +637,7 @@ TextBox.propTypes = {
   generated: PropTypes.bool,
   height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
   id: PropTypes.string,
+  inputProps: PropTypes.objectOf(PropTypes.any),
```

Comment:
In which case are we going to use this props ?

Reply:
That was requested in CNS-3912, to reduce the input height.